README.md

```diff
@@ -1 +1,18 @@
 # JS-Script
+
+## Getting Started
+
+This is a JavaScript script that let you create a file and write text into that particular file.
+
+### Prerequisites
+
+You need to have [NodeJS](https://nodejs.dev/en/) installed on your system
+
+### Usage
+
+1. Go to your file directory from your terminal.
```

Comment:
This is great, but I'd suggest that your README is too verbose for such a simple script. Can you boil this whole file down to just a few lines? Think about having to maintain it forever.

Specific note: dont need this line at all.